Composer/packages/client/src/pages/design/DesignPage.tsx

```diff
@@ -886,11 +886,10 @@ const DesignPage: React.FC<RouteComponentProps<{ dialogId: string; projectId: st
         {dialogId && (
           <CreateQnAModal dialogId={dialogId} projectId={projectId} qnaFiles={qnaFiles} onSubmit={handleCreateQnA} />
         )}
-        {displaySkillManifest && (
+        {displaySkillManifestOnProjectId && (
           <DisplayManifestModal
-            projectId={projectId}
-            skillNameIdentifier={displaySkillManifest}
-            onDismiss={() => dismissManifestModal(projectId)}
+            projectId={displaySkillManifestOnProjectId}
```

Comment:
why do we need a selector here. Cant it be just `projectId={skillId ?? projectId}` instead `projectId={displaySkillManifestOnProjectId}`

Reply:
This `displaySkillManifestOnProjectId` is from click project tree context menu, and skillId is from url
/bot/<botId>/skill/<skillId>

and when current workspace have multiple bot, if any of it request `create/edit manifest`, the modal should be display. that's what this selector do.

I thought about use local state just like in this page other `exportSkillModalInfo` `triggerModalInfo`, but the project tree click is not the only entrance of display it, so it's better to keep a global state. then it need a selector.